IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -94,11 +94,13 @@ - (instancetype)initWithDataSource:(id<MSIDExtendedTokenCacheDataSource>)dataSou
 
         for (MSIDCredentialCacheItem *cacheItem in results)
         {
+            MSID_LOG_WITH_CTX(MSIDLogLevelVerbose, context, @"(MSIDCredentialCache) filtering check on cached item with the following properties: client ID: %@, secret: %@, target: %@, realm: %@, environment: %@, familyID: %@, homeAccountId: %@, enrollmentId: %@, speInfo: %@, appKey: %@, applicationIdentifier: %@, tokenType: %@, kid: %@",cacheItem.clientId, cacheItem.secret, cacheItem.target, cacheItem.realm, cacheItem.environment, cacheItem.familyId, cacheItem.homeAccountId, cacheItem.enrollmentId, cacheItem.speInfo, cacheItem.appKey, cacheItem.applicationIdentifier, cacheItem.tokenType, cacheItem.kid);
```

Comment:
Please make following updates:
1. Don't log secret (it should never be logged since it's the actual token value)
2. Wrap homeAccountId with PII macros
3. Don't log appKey directly - use hashing
4. Application identifier is OII, use one of the PII macros

Reply:
Noted.

Reply:
Since PII masking might actually defeat the purpose of identifying the parameter that is different across multiple refresh tokens, I think it would be best to actually determine the number of unique instances of each value and then print it out.